packages/vats/src/core/chain-behaviors.js

```diff
@@ -146,8 +147,22 @@ export const bridgeProvisioner = async ({
       switch (obj.type) {
         case 'PLEASE_PROVISION': {
           const { nickname, address, powerFlags } = obj;
-          return E(provisioning)
-            .pleaseProvision(nickname, address, powerFlags)
+          let provisionP;
+          if (powerFlags.includes(PowerFlags.SMART_WALLET)) {
+            // Only provision a smart wallet.
+            provisionP = E(bridgeManager).toBridge(
+              BRIDGE_ID.PROVISION_SMART_WALLET,
+              obj,
+            );
+          } else {
```

Comment:
this is assuming the if `SMART_WALLET` isn't in the powerFlags that it should do this. why not validate the contents? I'm thinking the final `else` should be an error for an undefined case

Reply:
Can you please be more specific?  What kind of validation do you expect, since the Golang side of things already enforces that `powerFlags` is an array of strings?